client/src/modules/Course/Styles/Course.module.css

```diff
@@ -167,6 +168,13 @@
   color: var(--clr-gray-300);
 }
 
+.select-container {
```

Comment:
Would love if both sorts/filters could be put into a flexbox, also so that they would both take up the same amount of space.
![image](https://github.com/user-attachments/assets/89c6fce5-3753-495f-bb85-943e5fbf4114)
It would look best if the filter and sort both ended at the same distance, so it looked more rectangular. By encapsulating them in a div together, it would also be helpful for our future design (which I think is to move the sort and filters to below the title)